### PR TITLE
DOC-72 changes

### PR DESCRIPTION
**Pull request by cybsecmaster**

### CODE REVIEW

**Comment by MaelDre** [comment]
How do you access to these guides ?
-> the links "activating license" seems to be broken in the menu:
image

**Comment by cybsecmaster** [comment]
Added the pages to navigation menu:

image

File: docs/thehive/images/installation/license7.png
Comment (MaelDre) [inline docs/thehive/images/installation/license7.png:1]: Maybe we should remove your personal informations just leave, 
"Welcome", and remove first name/last name on top header bar

File: docs/thehive/images/installation/paid-license-1.png
Comment (MaelDre) [inline docs/thehive/images/installation/paid-license-1.png:1]: Same thing, it's better if we can remove my name on the top :)

File: docs/thehive/installation/activate-paid-license.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+In addition to the [Community License](./request-community-license.md), The Hive provides two advanced licensing options tailored to meet various operational requirements.
+
+### Gold License
+The Gold License is designed for internal response teams who benefit from enhanced case management lifecycles and integration within their organizational infrastructure.
+
+### Platinum License
+The Platinum License is best suited for large teams needing extensive server connections, clustering, and dedicated, continuous support for mission-critical, large-scale incident response operations.
+
+For full details, refer to [The Hive Pricing](https://strangebee.com/thehive-pricing-on-prem/).
```
Comment (MaelDre) [inline docs/thehive/installation/activate-paid-license.md:13]: TheHive (without space)

File: docs/thehive/installation/request-community-license.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# Request a Free Community License
+
+## Overview
+
+Starting with TheHive version 5.3, all users are required to generate a valid license to use the platform. Community users can obtain a **free Community License**, while paid licenses (Gold or Platinum) are available for enhanced functionality.
```
Comment (MaelDre) [inline docs/thehive/installation/request-community-license.md:5]: functionnalities ? (I'm not sure)

File: docs/thehive/installation/activate-paid-license.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Activating Paid License
+
+## Overview
+
+In addition to the [Community License](./request-community-license.md), The Hive provides two advanced licensing options tailored to meet various operational requirements.
```
Comment (MaelDre) [inline docs/thehive/installation/activate-paid-license.md:5]: TheHive (without space)

**Review by MaelDre — CHANGES_REQUESTED** [review]

File: docs/thehive/images/installation/license7.png
Comment (cybsecmaster) [reply docs/thehive/images/installation/license7.png:1]: This has been completed

File: docs/thehive/images/installation/paid-license-1.png
Comment (cybsecmaster) [reply docs/thehive/images/installation/paid-license-1.png:1]: This has been completed

File: docs/thehive/installation/activate-paid-license.md (hunk 1 above)
Comment (cybsecmaster) [reply docs/thehive/installation/activate-paid-license.md]: This has been completed

File: docs/thehive/installation/request-community-license.md (hunk 2 above)
Comment (cybsecmaster) [reply docs/thehive/installation/request-community-license.md]: This has been completed

File: docs/thehive/installation/activate-paid-license.md (hunk 3 above)
Comment (cybsecmaster) [reply docs/thehive/installation/activate-paid-license.md]: This has been completed

**Review by MaelDre — APPROVED** [review]

**Comment by MaelDre** [comment]
It looks good to me, but it seems there is a conflict with image directory

**Pull request merged**